paper-time-picker.html

```diff
@@ -374,9 +382,14 @@
           }
           this.rawValue = rawValue;
         },
-        _updateTime: function(hour, minute, second) {
+        _updateTime: function(hour, minute, second, enableSeconds) {
+          if (hour === undefined || minute === undefined ||
```

Comment:
add `|| enableSeconds === undefined` as well

Reply:
If this was the most recent comment, I applied this check right below this statement :-)